ui_components/src/sidebar/Sidebar.js

```diff
@@ -131,8 +131,8 @@ function Sidebar({ links, logoutTimer, navigationAPIData, userAPIData }) {
         return (
             <>
                 <div className="sb_avatar_container">
-                    <div className="sb_avatar_wrapper">
-                        <img className="sb_avatar_img" src={playerData.avatar_link}/>
+                    <div className={playerData.avatar_size < 200 ? "sb_avatar_wrapper_default" : "sb_avatar_wrapper_large"}>
+                        <img className={playerData.avatar_size < 200 ? "sb_avatar_img_default" : "sb_avatar_img_large"} src={playerData.avatar_link}/>
```

Comment:
The only difference between default and large is the size - For maximum flexibility here, how about leaving off the width from the regular stylesheet and declaring inline using `playerData.avatar_size`?

```suggestion
                    <div className="sb_avatar_wrapper">
                        <img className="sb_avatar_img" style={{ maxWidth: playerData.avatar_size, maxHeight: playerData.avatar_size}} src={playerData.avatar_link}/>
```